manifests/timer.pp

```diff
@@ -0,0 +1,99 @@
+# Create a timer and optionally a service unit
+# to execute with the timer unit.
+#
+# @api public
+#
+# @see systemd.timer
```

Comment:
Was this just a placeholder? Is this supposed to link to an external website?

Reply:
I copied it from the other types:

```
tmpfile.pp:# @see systemd-tmpfiles(8)
tmpfiles.pp:# @see systemd-tmpfiles(8)
unit_file.pp:# @see systemd.unit(5)
```

I added the (5) now.

Now if they she be there at all is another matter.

Reply:
@alexjfisher  remove it?

Reply:
Or link to eg. https://www.freedesktop.org/software/systemd/man/systemd.timer.html ??